CMakeLists.txt

```diff
@@ -34,17 +40,22 @@ if(APPLE)
 	find_library(CORE_MEDIA CoreMedia)
 	find_library(CORE_VIDEO CoreVideo)
 	find_library(CORE_SERVICES CoreServices)
+	find_library(SECURITY Security)
+	find_library(LDAP LDAP)
 
 	target_link_libraries(${PROJECT_NAME} PUBLIC
 		${APPLICATION_SERVICES}
 		${AUDIO_TOOLBOX}
 		${AV_FOUNDATION}
 		${CORE_AUDIO}
 		${CORE_FOUNDATION}
+		${SECURITY}
+		${LDAP}
```

Comment:
Why LDAP??

Reply:
I don't remember, but do we need something to manage the login in the project?

Reply:
This is just a demo project for libmediasoupclient. There is no place for custom changes.

You should create your own project if it requires any particularities.

Reply:
I did not change the code, only the cmake, it is not my personal demo project.
I don't remember why I need ldap as dependency, let me study a little bit the application